docs/cache/getting-started.mdx

```diff
@@ -0,0 +1,107 @@
+---
+sidebar_position: 3
+sidebar_label: Getting Started
+title: Getting Started
+pagination_prev: null
+pagination_next: null
+description: Jump in and get started quickly using Momento Cache with your SDK of choice.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Getting started with Momento Cache
+
+If you're looking to jump in and get started with Momento Cache with an SDK, you've come to the right place.
+
+If you want to quickly get a feel for the Momento Cache experience without installing any software? You can do so right here [in your browser](#try-our-cli-and-an-sdk-in-your-browser).
```

Comment:
Should this go in the "Using the Momento Console" section?

Reply:
idk if that #try-our-cli anchor is still valid. We used to have this demo that allowed you to run the CLI in your browser window but we don't support that anymore, so this link might be dead.